### ISSUE TITLE
Catch importlib.util.find_spec returning None

### ISSUE THREAD

**Issue by OzRamsay** [opsdroid/opsdroid#440]
# Description

Failed initialisation of freshly installed opsdroid leads to a signaling error

## Steps to Reproduce
No changes has been made to the configuration fail.

## Experienced Functionality
```yaml
Traceback (most recent call last):
  File "/usr/local/bin/opsdroid", line 11, in <module>
    load_entry_point('opsdroid==0.11.2', 'console_scripts', 'opsdroid')()
  File "/usr/local/lib/python3.6/dist-packages/opsdroid/__main__.py", line 130, in main
    opsdroid.start_loop()
  File "/usr/local/lib/python3.6/dist-packages/opsdroid/core.py", line 136, in start_loop
    self.loader.load_modules_from_config(self.config)
  File "/usr/local/lib/python3.6/dist-packages/opsdroid/loader.py", line 257, in load_modules_from_config
    skills = self._load_modules('skill', config['skills'])
  File "/usr/local/lib/python3.6/dist-packages/opsdroid/loader.py", line 313, in _load_modules
    module = self.import_module(config)
  File "/usr/local/lib/python3.6/dist-packages/opsdroid/loader.py", line 58, in import_module
    module = Loader.import_module_from_spec(module_spec)
  File "/usr/local/lib/python3.6/dist-packages/opsdroid/loader.py", line 37, in import_module_from_spec
    module_spec.loader.exec_module(module)
AttributeError: 'NoneType' object has no attribute 'exec_module'
Exception ignored in: <bound method BaseEventLoop.__del__ of <_UnixSelectorEventLoop running=False closed=True debug=False>>
Traceback (most recent call last):
  File "/usr/lib/python3.6/asyncio/base_events.py", line 512, in __del__
  File "/usr/lib/python3.6/asyncio/unix_events.py", line 65, in close
  File "/usr/lib/python3.6/asyncio/unix_events.py", line 146, in remove_signal_handler
  File "/usr/lib/python3.6/signal.py", line 47, in signal
TypeError: signal handler must be signal.SIG_IGN, signal.SIG_DFL, or a callable object
```

## Versions
- **Opsdroid version:** master
- **Python version:** 3.6
- **OS/Docker version:** Lubuntu 17.10

## Configuration File
Default configuration

**Comment by jacobtomlinson** [comment]
Duplicate of #298

**Comment by jacobtomlinson** [comment]
The signal error is not the real problem here. That is just an annoying bug which pops up. The actual issue is a few lines above in the traceback.

```
AttributeError: 'NoneType' object has no attribute 'exec_module'
```

**Comment by jacobtomlinson** [comment]
@OzRamsay are you sure you're using the default config?

**Comment by OzRamsay** [comment]
The case is that I did receive errors without changing the config (as I reinstalled the OS wiping all the data, I am sure about that). However, sometimes it did work, so eventually, I did change the config. Then I reinstalled the Opsdroid, but I didn't delete .opsdroid folder. So I assume the previous problem was because of the misconfiguration. 
I did not save the traceback of errors that I received after reinstalling the OS (and before changing the config), It looked similar to this one, but actually it could differ from the traceback I provided to this issue.    

So now I see that it could be a false alarm
